cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,86 @@
+= CloudHub VPC to CloudHub 2.0 Private Space Upgrade  
+
+//Overview
+To migrate your workloads from CloudHub 1.0 to CloudHub 2.0, you have to start by setting up your CloudHub 2.0 infrastructure and network connections. One way to achieve this is by creating fresh CloudHub 2.0 Private Spaces with different configurations from what you had on CloudHub 1.0. However, if you prefer to keep your infrastructure configuration the same, you can use the in-product upgrade tool to easily upgrade your CloudHub dedicated VPC to a CloudHub 2.0 private space.
+Once your new private space is ready, you can gradually migrate your applications from the CloudHub VPC to the CloudHub 2.0 private space. After all your applications have been successfully moved, you can decommission your old VPC.
+
+
+// Benefits
+This VPC to private space in-product upgrade tool helps you speed up the creation of a CloudHub 2.0 private space. Instead of starting from scratch, you can easily upgrade your existing CloudHub dedicated VPC. This process saves you time by eliminating the need to acquire new CIDR blocks, set up new network connections, and configure the private space from the ground up.
+The tool provisions the infrastructure for a CloudHub 2.0 private space by cloning the existing configurations of your eligible CloudHub dedicated VPC, using the same CIDR block. If your VPC has existing TGW or VPN network connections, they are seamlessly transferred to the new CloudHub 2.0 private space. This ensures that any new applications deployed into the new private space can communicate effectively.
```

Comment:
We should also add this: The default Cloudhub certs will be configured as part of the private space's default certificates, and the custom domains from the DLB(s) of the migrated VPC will be created as part of the private space's custom TLS certificates.